### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by SimonHeybrock**

### CODE REVIEW

File: docs/about/release-notes.rst [hunk 1]
```diff
@@ -3,29 +3,48 @@
...
+Contributors
+~~~~~~~~~~~~
+
+v0.5 (January 2021)
+-------------------
+
+Features
+~~~~~~~~
+
+* New ``profile`` plotting functionality where one of the slider dimensions can be displayed as a profile in a subplot (1-D and 2-D projections only).
```
Comment (OwenArnold) [inline docs/about/release-notes.rst:24]: Should we consider making subsection for plotting?

Comment (OwenArnold) [inline docs/about/release-notes.rst:24]: Could make this briefer and just link to the relevant docs https://scipp.github.io/visualization/plotting-nd-data.html#Profile-picking

File: docs/about/release-notes.rst [hunk 2]
```diff
@@ -3,29 +3,48 @@
...
 * Sliders have a thickness slider associated with them and can be used to show slices of arbitrary thickness.
 * Can hide/show individual masks on plots.
 * Can toggle log scale of axes and colorbar with buttons in figure toolbar.
 * Add binned data support, replacing "event list" dtypes as well as "realign" support.
+* Plotting of event data (binned data) with dynamic resampling with "infinite zoom" functionality.
 * Value-based slicing support.
-* Support for saving scipp objects to HDF5.
 * Possibility to plot Scipp objects using ``my_data_array.plot()`` in addition to the classical ``plot()`` free function.
+* Support for saving and loading scipp data structures to HDF5.
```
Comment (OwenArnold) [inline docs/about/release-notes.rst:32]: https://scipp.github.io/user-guide/reading-and-writing-files.html?highlight=hdf5

**Review by OwenArnold — APPROVED** [review]
Generally good.

I would suggest the use of links where possible so that readers can be taken directly through to relevant sections on the docs. Could be done as a follow up if needed.

**Comment by SimonHeybrock** [comment]
> Generally good.
> 
> I would suggest the use of links where possible so that readers can be taken directly through to relevant sections on the docs. Could be done as a follow up if needed.

As previously discussed, I don't think this is a good idea, because everty time we change the structure of documentation we need to go back to *all* old release notes and fix links.

File: docs/about/release-notes.rst (hunk 1 above)
Comment (SimonHeybrock) [reply docs/about/release-notes.rst:24]: See below.

File: docs/about/release-notes.rst (hunk 2 above)
Comment (SimonHeybrock) [reply docs/about/release-notes.rst:32]: See below.

**Comment by OwenArnold** [comment]
> > Generally good.
> > I would suggest the use of links where possible so that readers can be taken directly through to relevant sections on the docs. Could be done as a follow up if needed.
> 
> As previously discussed, I don't think this is a good idea, because everty time we change the structure of documentation we need to go back to _all_ old release notes and fix links.

Not necessarily. We have version information in the release notes https://scipp.github.io/release/0.4.0/user-guide/masking.html

**Comment by SimonHeybrock** [comment]
> > > Generally good.
> > > I would suggest the use of links where possible so that readers can be taken directly through to relevant sections on the docs. Could be done as a follow up if needed.
> > 
> > 
> > As previously discussed, I don't think this is a good idea, because everty time we change the structure of documentation we need to go back to _all_ old release notes and fix links.
> 
> Not necessarily. We have version information in the release notes https://scipp.github.io/release/0.4.0/user-guide/masking.html

Unless you are proposing that the documentation should only ever contain the latest release notes, and all notes from previous releases should be removed from the page, this is irrelevant?

**Comment by SimonHeybrock** [comment]
... or at least some weird, chick-egg-problem?

**Pull request merged**